### PR TITLE
DISTMYSQL-154 Release notes PDPXC 8.0.27

### PR DESCRIPTION
**Pull request by nastena1606**

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
LGTM

File: source/release-notes-pxc-v8.0.27.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+|pdmysql| is a single solution with the best and most critical enterprise components from the |mysql| open source community, designed and tested to work together. 
+
+|pdmysql| provides two deployment variants: one is based on |PS| and another one is based on |PXC|. 
+
+This release of |pdmysql| is focused on the |PXC|-based deployment variant. It is based on `Percona XtraDB Cluster 8.0.27-17.1 <https://www.percona.com/doc/percona-xtradb-cluster/8.0/release-notes/Percona-XtraDB-Cluster-8.0.27-17.1.html>`_ 
+
+Release Highlights
+==================
+
+The following lists a number of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
```
Comment (LenzGr) [inline source/release-notes-pxc-v8.0.27.rst:19]: Would it make sense to include a link to the upstream MySQL 8.0.27 release notes here? This way, users could quickly check what other upstream changes have occurred, without having to check the PS 8.0.27 release notes first.

Comment (nastena1606) [reply source/release-notes-pxc-v8.0.27.rst:19]: Added

**Review by LenzGr — APPROVED** [review]
LGTM, thank you for considering my improvement suggestion!

**Review by george-lorch — APPROVED** [review]

**Pull request merged**